src/ecosystemIndex.jsx

```diff
@@ -583,6 +583,12 @@ async function init() {
                     path='/not-found'
                     component={NotFound}
                   />
+                  {userAccessToSite?.enabled && (
+                    <Route
+                      path={userAccessToSite.deniedPageURL || '/access-denied'}
+                      component={AccessDenied}
```

Comment:
you need to import `AccessDenied` and `userAccessToSite` before using them
this also applies to other bundle homepages